technology/protocols.md

```diff
@@ -58,7 +58,7 @@ setTokens([ tokenContractAddress, ... ])
 
 # RequestForQuoteERC20
 
-AirSwap [RequestForQuoteERC20](./glossary.md#request-for-quote-rfq) is a client-server protocol used by market makers running servers from which clients request ERC20 orders via HTTP or WebSocket. In RFQ, the server is the signer (i.e. `signerAmount`, `signerToken`) and the client is the sender (i.e. `senderAmount`, `senderToken`).
+AirSwap [AirSwap RequestForQuoteERC20](./glossary.md#request-for-quote-rfq) is a client-server protocol used by market makers running servers from which clients request ERC20 orders via HTTP or WebSocket. In RFQ, the server is the signer (i.e. `signerAmount`, `signerToken`) and the client is the sender (i.e. `senderAmount`, `senderToken`).
```

Comment:
Redundant "AirSwap"

Reply:
Yeah. I think the link was broken because /glossary is 1 directory up. Changed the path